lib/templates/json_texture.template.js

```diff
@@ -0,0 +1,43 @@
+path = require("path");
+
+function jsonTextureTemplate(data) {
+  var spriteObj = {};
+
+  // Create frame data for each sprite.
+  spriteObj.frames={};
+
+  data.sprites.forEach(function (sprite) {
+    var entry={
+      frame: {
+        "x": sprite.x,
+        "y": sprite.y,
+        "w": sprite.width,
+        "h": sprite.height
+      }
+    };
+
+    spriteObj.frames[path.basename(sprite.source_image)]=entry;
+  });
+
+  // Create the meta data.
+  spriteObj.meta = {
+    "app": "spritesheet-templates",
+    "version": "9.5.0",
+    "image": path.basename(data.spritesheet.image),
+    "format": "RGBA8888",
```

Comment:
Is there ever a time when we wouldn't have an alpha channel nor 32-bit sprite? (e.g. `png-8`, `jpeg`)